updates/stable.json

```diff
@@ -25,7 +25,7 @@
       "metadata": {
         "rollout": {
           "duration_minutes": 2880,
-          "start_epoch": 1569249000,
+          "start_epoch": 1600871400,
```

Comment:
stable rollout of 32.20200907.3.0 scheduled for 2020-09-23 14:30:00 UTC for 2880m (48.0h)